.github/workflows/build-and-deploy.yml

```diff
@@ -17,26 +17,29 @@ concurrency:
 jobs:
```

Comment:
![image](https://github.com/user-attachments/assets/ff0acd37-298d-4ddc-8ea3-8cba728a3079)
`build`  job has been verified
`depoly` is expected to fail due to branch restriction.

Reply:
https://github.com/coinbase/coinbase-wallet-sdk/actions/runs/10893719985